scripts/setup.bat

```diff
@@ -0,0 +1,21 @@
+:: Install the venv virtual environment
+@echo off
+
+echo Setting up aggregator environment...
+cd ./aggregator
```

Comment:
You are using relative paths but is it not possible that this script could be ran from the scripts folder(or anywhere else)? The BAT script should probably ensure that it is in the correct directory before referring to relative ones.